File: JavaBasic/138_count_vowels.java
// 138	138_count_vowels.java	Count the number of vowels in a given string using a loop.
public class _138_count_vowels {
    public static void main(String[] args) {
        String str = "Hello World";
        int count = 0;
        String vowels = "aeiouAEIOU";
        for (char c : str.toCharArray()) if (vowels.indexOf(c) != -1) count++;
        System.out.println("String: " + str + " | Vowels: " + count);
    }
}

File: JSBASIC/138_count_vowels.js
// 138	138_count_vowels.js	Count the number of vowels in a given string using a loop.

let str = "Hello World";
let count = 0;
let vowels = "aeiouAEIOU";
for (let i = 0; i < str.length; i++) {
    if (vowels.includes(str[i])) count++;
}
console.log("String:", str);
console.log("Number of vowels:", count);
